app/views/accounts/_webauthn_roaming.html.erb

```diff
@@ -19,13 +10,22 @@
       </div>
       <% if MfaPolicy.new(current_user).multiple_factors_enabled? %>
         <div class="grid-col-4 mobile-lg:grid-col-6 right-align">
-          <%= link_to(
-                t('account.index.webauthn_delete'),
-                webauthn_setup_delete_path(id: cfg.id),
-              ) %>
+          <%= render ButtonComponent.new(
+                action: ->(**tag_options, &block) do
+                  link_to(webauthn_setup_delete_path(id: cfg.id), **tag_options, &block)
+                end,
+              ).with_content(t('account.index.webauthn_delete')) %>
```

Comment:
I don't think we need these changes to use button component here? Or at least, I would expect it to be unstyled.

![image](https://user-images.githubusercontent.com/1779930/154744622-d506a985-a7a9-4cd6-b042-e808e15e28b9.png)

Compare to:

![image](https://user-images.githubusercontent.com/1779930/154744682-f045a468-34b7-4361-8b92-d34e960da5c8.png)